tests/integration/targets/inventory_aws_rds/runme.sh

```diff
@@ -15,33 +15,55 @@ export ANSIBLE_INVENTORY=test.aws_rds.yml
 # test empty inventory config
 ansible-playbook playbooks/test_invalid_aws_rds_inventory_config.yml "$@"
 
+# delete existing resources
+ansible-playbook playbooks/setup_instance.yml -e "operation=delete" -e "aws_api_wait=true" "$@"
+
+{
+
 # generate inventory config and test using it
-ansible-playbook playbooks/create_inventory_config.yml "$@"
-ansible-playbook playbooks/test_populating_inventory.yml "$@"
+ansible-playbook playbooks/create_inventory_config.yml "$@" &&
```

Comment:
If I understand correctly, the `&&` are used to be able to clean up in case of problem.

Since we've got already the `set -eux` in place, you can do the same thing with a `trap` and a clean up function. Here is an example https://gist.github.com/goneri/9b801be15da37e7cee44295608bf1000